src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -96,6 +97,7 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
   const isLoadingSchedule = isLoading || isEventsLoading;
 
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
+  const [scheduleScaleFactor, setScheduleScaleFactor] = useState(1);
```

Comment:
This should be using the constant with all of the factors.

`const DEFAULT_SCHEDULE_FACTOR = SCHEDULE_SCALE_FACTORS[1]`